client/my-sites/checkout/composite-checkout/payment-method-processors.js

```diff
@@ -90,7 +90,7 @@ export function applePayProcessor( submitData ) {
 	return pending;
 }
 
-export async function stripeCardProcessor( submitData, isLoggedOutCart ) {
+export async function stripeCardProcessor( submitData, createUserAndSiteBeforeTransaction ) {
```

Comment:
This all looks fine, although I wonder if it might be worth adding an `options` object argument to each of these functions (this one and all the others) instead of the boolean because I think that this:

```js
stripeCardProcessor( { name: 'person' }, { createUserAndSiteBeforeTransaction: true } );
```

is more readable than this:

```js
stripeCardProcessor( { name: 'person' }, true );
```

Reply:
I've implemented this change in this PR.